src/uu/stty/circumflex-control-characters.txt

```diff
@@ -0,0 +1,240 @@
+
```

Comment:
if it is generated at build time, does it need to be in tree ?

and if the answer is yes, could you please add a comment on top of this file to explain what it is, thanks

Reply:
@andrewliebenow ping ? :)